insns.def

```diff
@@ -350,6 +350,7 @@ putspecialobject
 (rb_num_t value_type)
 ()
 (VALUE val)
+// attr bool leaf = (value_type != VM_SPECIAL_OBJECT_CBASE); /* get cbase may allocate a singleton */
```

Comment:
allocation violate the `leaf` assumption?

Reply:
Ah maybe it does not I was mistaken, but I think this can also raise in the case of special constant so can't be leaf. I will update the comment.